### DISCUSSION TITLE
build.ps1 error

### DISCUSSION THREAD

**Discussion by github-delusion** [Qonfused/OSX-Hyper-V#18] [Q&A]
Hello, 

I have error when using bulid.ps1

Please, help me. What I doing bad?

Thank you

OSX-Hyper-V\scripts> .\build.ps1
Downloading OCE Build CLI...

Start-Process : This command cannot be run due to the error: %1 is not a valid Win32 application.
At line:46 char:1
+ Start-Process -Wait $BINARY_PATH -NoNewWindow -ArgumentList "build -c ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperationException,Microsoft.PowerShell.Commands.StartProcessCommand

**Comment by Qonfused** [comment]
It seems like you're using an older version of Powershell that doesn't provide the `$IsWindows` Powershell variable. I've patched the bootstrapper script in https://github.com/Qonfused/OCE-Build/commit/81c313747806af7cabe8cd536243f8ae925593b4 to set these variables as a fallback which should no longer raise this issue.

The build should work without any changes to the OSX-Hyper-V project, so you can re-run the build on your current clone.
